Error with the docker image

I can't play with it :
```
$ docker run -it -p 8080:8080 samdotli/rendertron

> bot-render@1.0.0 start /app
> node src/main.js

{ Error: connect ECONNREFUSED 127.0.0.1:44240
    at Object._errnoException (util.js:1041:11)
    at _exceptionWithHostPort (util.js:1064:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1153:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 44240 }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bot-render@1.0.0 start: `node src/main.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bot-render@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/botrender/.npm/_logs/2017-09-05T22_48_53_496Z-debug.log
```

Same problem with me.
I also try running with SECCOMP and `cap-add SYS_ADMIN` but it did not work.

I built image from master (a335926a).

My docker info:
```
Server Version: 17.07.0-ce
Storage Driver: overlay
 Backing Filesystem: extfs
 Supports d_type: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 3addd840653146c90a254301d6c3a663c7fd6429
runc version: 2d41c047c83e09a6d61d464906feb2a2f3c52aa4
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 3.10.0-514.26.2.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 11.33GiB
```

Same error with both 2 commands:
```docker run -it --rm -p 8080:8080 --cap-add SYS_ADMIN --name rendertron-container rendertron```
```docker run -it -p 8080:8080 --security-opt seccomp=/opt/rendertron/chrome.json --name rendertron-container rendertron```

Container logs:
```
> bot-render@1.0.0 start /app
> node src/main.js

{ Error: connect ECONNREFUSED 127.0.0.1:40414
    at Object._errnoException (util.js:1041:11)
    at _exceptionWithHostPort (util.js:1064:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1153:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 40414 }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bot-render@1.0.0 start: `node src/main.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bot-render@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/botrender/.npm/_logs/2017-09-08T08_25_56_104Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -166,7 +166,7 @@ Start a container with the built image using SYS_ADMIN:
 docker run -it -p 8080:8080 --cap-add SYS_ADMIN --name rendertron-container rendertron
 ```
 
-To check if your kernel is compatible with Docker, follow [Docker's instructions](https://docs.docker.com/engine/installation/linux/linux-postinstall/#troubleshooting). For CentOS 7, which doesn't have user namespaces enable it, you will [need to enable them](https://github.com/GoogleChrome/rendertron/issues/96#issuecomment-328305721).
+To check if your kernel is compatible with Docker, follow [Docker's instructions](https://docs.docker.com/engine/installation/linux/linux-postinstall/#troubleshooting). For CentOS 7, which doesn't have user namespaces enabled, you will [need to enable them](https://github.com/GoogleChrome/rendertron/issues/96#issuecomment-328305721).
 
 ### Using the container
 Load the homepage in any browser:

--- a/README.md
+++ b/README.md
@@ -150,6 +150,7 @@ Building the container:
 docker run -it -p 8080:8080 --name rendertron-container rendertron
 ```
 
+### Connection error (ECONNREFUSED)
 In the case where your kernel lacks user namespace support or are receiving a `ECONNREFUSED` error when trying to access the service in the container (as noted in issues [2](https://github.com/GoogleChrome/rendertron/issues/2) and [3](https://github.com/GoogleChrome/rendertron/issues/3)), the two recommended methods below should solve this:
 1. [Recommended] - Use [Jessie Frazelle' seccomp profile](https://github.com/jessfraz/dotfiles/blob/master/etc/docker/seccomp/chrome.json) and `-security-opt` flag
 2. Utilize the `--cap-add SYS_ADMIN` flag
@@ -165,6 +166,9 @@ Start a container with the built image using SYS_ADMIN:
 docker run -it -p 8080:8080 --cap-add SYS_ADMIN --name rendertron-container rendertron
 ```
 
+To check if your kernel is compatible with Docker, follow [Docker's instructions](https://docs.docker.com/engine/installation/linux/linux-postinstall/#troubleshooting). For CentOS 7, which doesn't have user namespaces enable it, you will [need to enable them](https://github.com/GoogleChrome/rendertron/issues/96#issuecomment-328305721).
+
+### Using the container
 Load the homepage in any browser:
 ```bash
 http://localhost:8080/

